app/src/main/java/edu/ucandroid/weathernotice/dto/LocationDetails.kt

```diff
@@ -0,0 +1,4 @@
+package edu.ucandroid.weathernotice.dto
```

Comment:
These three file changes were added in order to finish the implementation of Location services. Since they are considered dangerous permissions, the user should be prompted for permission to access. Obivously, this is just boilerplate and doesn't have any implementation quite yet, but can serve as a starting point for how you guys choose to implement it.

https://www.youtube.com/watch?v=VgZZemAwLTk&feature=youtu.be